File: app/src/main/java/com/example/vidya_vahinitransportationassistance/models/BusStop.java
package com.example.vidya_vahinitransportationassistance.models;

public class BusStop {
    private String name;
    private double latitude;
    private double longitude;
    private int travelTime;

    public BusStop(String name, double latitude, double longitude, int travelTime) {
        this.name = name;
        this.latitude = latitude;
        this.longitude = longitude;
        this.travelTime = travelTime;
    }

    public String getName() { return name; }
    public double getLatitude() { return latitude; }
    public double getLongitude() { return longitude; }
    public int getTravelTime() { return travelTime; }
}

File: app/src/main/java/com/example/vidya_vahinitransportationassistance/data/model/BusStop.kt
package com.example.vidya_vahinitransportationassistance.data.model

/**
 * Data class representing a bus stop with geographical coordinates.
 */
data class BusStop(
    val name: String,
    val latitude: Double,
    val longitude: Double,
    val standardTravelTimeMinutes: Int = 0
)
